lib/deployment_runner.py

```diff
@@ -141,7 +141,7 @@ def content_provider(self, page):
 
     def subscription_management(self, page):
         '''Subscription Management Application'''
-        page.click_sma_radio(self.sat.rhsm_satellite_name)
+        page.click_sma_radio_by_uuid(self.sat.rhsm_satellite['uuid'])
```

Comment:
I think this as been requested in the past but since these are dynamically generated radio buttons based on the number of SMAs returned by the customer portal api they are given a generic name plus index number.  The element attributes have the unique info we need so that we don't have to do a crazy search for a value then backtracking up the element tree.